packages/api-server/controllers/auth.js

```diff
@@ -0,0 +1,118 @@
+require('dotenv').config();
+
+import jwt from 'jsonwebtoken';
+import {Customer,Identity} from '../models';
+import {comparePassword, getHashedPassword} from '../utils/password';
+import multer from 'multer';
+
+// SET STORAGE
+var storage = multer.diskStorage({
+    destination: function (req, file, cb) {
+      cb(null, 'public')
+    },
+    filename: function (req, file, cb) {
+      cb(null, file.fieldname + '-' + Date.now())
+    }
+});
+   
+var upload = multer({ storage: storage }).fields([{name: "front_image"}, {name: "back_image"}]);
+
+const uploadImage = async (req,res,next)=>{
+    upload(req,res,err=>{
+        if (err) return res.json({ message: err.message });
+        if (!req.files.front_image) return res.json({ message: "Please upload front image" });
+        if (!req.files.back_image) return res.json({ message: "Please upload back image" });
+
+        req.body.front_image = req.files.front_image;
+        req.body.back_image = req.files.back_image;
+        next();
+    })
+}
+
+
+const register = async(req,res)=>{ 
+    let {fullname, birthday, phone_number, password, email, address,
```

Comment:
use `const` instead of `let` and create another variable instead of changing value directly